tests/run/cpp_stl_atomic.pyx

```diff
@@ -0,0 +1,106 @@
+# distutils: extra_compile_args=-std=c++11
+# mode: run
+# tag: cpp, werror
+
+from cython.operator cimport preincrement as incr
+from libc.stdint cimport *
+
+from libcpp.atomic cimport atomic
+
+def int_test(int x):
+    """
+    >>> int_test(55)
+    3
+    >>> int_test(42)
+    3
+    >>> int_test(100000)
+    3
+    """
+    atom = new atomic[int](x)
+    try:
+        atom.store(0)
+        incr(atom)
```

Comment:
I could be wrong but `atom` is a pointer. Doesn't `incr` just change the pointer? Replace it with `incr(atom[0])` I think.